tools/frr-reload.py

```diff
@@ -1961,6 +1962,44 @@ def compare_context_objects(newconf, running):
     return (lines_to_add, lines_to_del)
 
 
+class LogFmtFormatter(logging.Formatter):
+    def format(self, record):
+        # Escape double quotes in the message, replace newlines with '\n'
+        record.msg = record.getMessage().replace('"', '\\"')
+        record.msg = record.msg.replace('\n', '\\n')
+        # Format the time in RFC 3339 format
+        record.asctime = datetime.datetime.fromtimestamp(record.created, datetime.timezone.utc).astimezone().isoformat(timespec='seconds')
+        # Create logfmt style log message, ignore extra fields
+        logfmt = f'ts={record.asctime} level={record.levelname} msg="{record.msg}"'
+        for key, value in record.__dict__.items():
+            if key in [
+                "args",
+                "asctime",
+                "created",
+                "exc_info",
+                "exc_text",
+                "filename",
+                "funcName",
+                "levelname",
+                "levelno",
+                "lineno",
+                "module",
+                "msecs",
+                "msg",
+                "name",
+                "pathname",
+                "process",
+                "processName",
+                "relativeCreated",
+                "stack_info",
+                "thread",
+                "threadName",
+            ]:
+                continue
+            logfmt += f" {key}={value}"
+        return logfmt
```

Comment:
```suggestion
        # Create logfmt style log message, ignore extra fields
        logfmt = f'ts={record.asctime} level={record.levelname} msg="{record.msg}"'
        extra_fields = [
            "args",
            "asctime",
            "created",
            "exc_info",
            "exc_text",
            "filename",
            "funcName",
            "levelname",
            "levelno",
            "lineno",
            "module",
            "msecs",
            "msg",
            "name",
            "pathname",
            "process",
            "processName",
            "relativeCreated",
            "stack_info",
            "thread",
            "threadName",
        ]
        for key, value in record.__dict__.items():
            if key in extra_fields:
                continue
            logfmt += f" {key}={value}"
        return logfmt
```

Personally, I think moving all the extra fields into their own list instead of putting them directly in the if statement makes this a bit easier to read.

Reply:
those are rather the undesired/default extra fields, but yes, maybe better in their own list